solve_test.go

```diff
@@ -47,6 +49,8 @@ func overrideIsStdLib() {
 	isStdLib = func(path string) bool {
 		return false
 	}
+	// NOTE(narqo): this is an ugly hack! One have to think about better way to do cross-package mocking.
+	pkgtree.MockIsStdLib()
```

Comment:
I really don't like this, but currently, I don't see another way to make fixtures work.